sources/HUBComponentWrapper.m

```diff
@@ -432,6 +432,11 @@ - (BOOL)gestureRecognizer:(UIGestureRecognizer *)gestureRecognizer shouldReceive
     UIView *currentView = touch.view;
     
     while (currentView != nil && currentView != self.view) {
+
+        if ([currentView isKindOfClass:[UISegmentedControl class]]) {
```

Comment:
Wondering if we should not add this check and instead change the `UIButton` check to `UIControl`?

@mhallendal @cerihughes What do you and CA think?